src/frequenz/sdk/microgrid/_data_pipeline.py

```diff
@@ -433,6 +471,28 @@ def logical_meter() -> LogicalMeter:
     return _get().logical_meter()
 
 
+def consumer() -> Consumer:
+    """Return the consumer instance.
```

Comment:
I would explicitly cross-reference here to the `Consumer` type, and probably just say "Returns a measuring point for the [consumer][...] part of the microgrid" or something like that.